README.md

```diff
@@ -16,8 +26,18 @@ Be sure to install node modules:
 npm install
 ```
 
-It is no ready to start a local development environment:
+It is now ready to start a local development environment:
 
 ```shell
 npm start
 ```
+
+## Authors
+
+- **Kyle Shevlin** - _Initial work_ - [Twitter](https://twitter.com/kyleshevlin)
+
+<!-- See also the list of [contributors](https://github.com/your/project/contributors) who participated in this project. -->
```

Comment:
Go ahead and uncomment this, make it:
```
## Contributors
See the list of contributors [here](https://github.com/SecondCareerDevs/website/contributors).
```

We'll make it better with photos and stuff later.